microservices/first/utils/db_manager.py

```diff
@@ -3,7 +3,7 @@
 from sqlite3 import Error
 from typing import Any, Dict, List, Optional, Union
 
-from microservices.first.utils import deployment_status
+import deployment_status
```

Comment:
Have you tested this? I think it needs to be `.deployment_status`
```suggestion
import .deployment_status
```

Reply:
Second this.